src/utils/simplifyAxios.ts

```diff
@@ -0,0 +1,38 @@
+import { AxiosError, AxiosResponse } from "axios";
+
+/**
+ * given a result of an ajax call, returns a promise that always resolves, see return type
+ * this mimics Future<Either<Err, Res>> composition (like in functional languages) that is more practical and conserves error type
+ * this helper function tries to simplify the usage of the methods generated by openapi-tools trying to strike a balance between type safety and ease of use
+ * types generated are unfortunately not complete and wrong in some cases
+ */
+type StatusCodeOk = 200 | 201 | 204;
+type StatusCodeKo = 400 | 403 | 404 | 409 | 404 | 500;
+export async function simplifyAxios<Data>(
```

Comment:
What do you think about renaming this function in `normalizeAxiosResponse`?

Reply:
done